lib/internal/Magento/Framework/Pricing/PriceCurrencyInterface.php

```diff
@@ -47,7 +47,7 @@ public function convertAndRound($amount, $scope = null, $currency = null, $preci
      * @param int $precision
      * @param null|string|bool|int|\Magento\Framework\App\ScopeInterface $scope
      * @param \Magento\Framework\Model\AbstractModel|string|null $currency
-     * @return float
+     * @return string
```

Comment:
Hi @lbajsarowicz, thanks for collaboration. Please declare strict return type according to [Magento technical guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html)